api/net/Models/Auth/PrincipalModel.cs

```diff
@@ -86,6 +86,11 @@ public class PrincipalModel
     /// </summary>
     public AccountAuthState AuthState { get; set; }
 
+    /// <summary>
+    /// get/set - An array of media types not accessible to this user.
+    /// </summary>
+    public IEnumerable<int> MediaTypes { get; set; } = Array.Empty<int>();
```

Comment:
You should also add the `SourceTypes` while you're here.  The subscriber is also limited to which sources they're supposed to be allowed to see.

Reply:
@Fosol - updated with sources!